src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowJob.java

```diff
@@ -188,7 +188,12 @@ public void setDefinition(FlowDefinition definition) {
         } else {
             quietPeriod = null;
         }
-        makeDisabled(json.optBoolean("disable"));
+        // TODO remove compatibility when baseline includes https://github.com/jenkinsci/jenkins/pull/6485
```

Comment:
Could you clarify in the comment which side is the compatibility side vs. the new side?

Reply:
I wouldn't worry too much about the comment, there's a PR to update the baseline to take advantage of the new APIs https://github.com/jenkinsci/workflow-job-plugin/pull/261 the TODO will be resolved there.